app/components/devnet/DevnetFaucetModal.tsx

```
@@ -0,0 +1,343 @@
+      <div
+        ref={modalRef}
+        className="mx-4 w-full max-w-md rounded-none border border-[var(--border)] bg-[var(--bg)] shadow-2xl"
+        role="dialog"
+        aria-modal="true"
+        aria-label="Fund your devnet account"
+      >
+        {/* Header */}
+        <div className="flex items-center justify-between border-b border-[var(--border)]/50 px-5 py-3">
+          <div>
+            <div className="text-[9px] font-medium uppercase tracking-[0.25em] text-[var(--accent)]/60">
+              // devnet
```

Biome (2.4.4)

[error] Wrap comments inside children within braces.

(lint/suspicious/noCommentText)